admin/login.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+session_start();
+const LOGINLIST = ['admin' => '$2y$10$i/I2uaPwmad5zPeSdpLSzuJpvgCbxk6YSuHlCjD7sE7BO116JflEW'];
+
+require('../functions/login.php');
+
+if (isset($_POST['user']) || isset($_POST['pass'])) {
+    header('Location: ' . verifyPassword($_POST['user'], $_POST['pass']));
```

Comment:
I think I would pass const LOGINLIST into this function - I appreciate it works without passing it in but it is not terribly readable.

Reply:
Done.